pkg/reconciler/ksvc/resources/utils.go

```diff
@@ -0,0 +1,21 @@
+package resources
```

Comment:
This package makes me miss my old hammer: https://lodash.com/docs/

Reply:
I hate to say -- does this class need tests?

Reply:
for sure, probably I wrote these utilities so many times so that I just assume it works :). Sadly Go does not have a standard libraries for all these simple utilities.